contracts/SportMarkets/Parlay/ParlayMarketsAMM.sol

```diff
@@ -498,24 +501,36 @@ contract ParlayMarketsAMM is Initializable, ProxyOwned, ProxyPausable, ProxyReen
         }
     }
 
-    function _storeRisk(
-        address[] memory _sportMarkets,
-        uint[] memory _positions,
-        uint _sUSDPaid
-    ) internal {
+    function _storeRisk(address[] memory _sportMarkets, uint _sUSDPaid) internal {
         if (_sportMarkets.length == 2) {
-            riskPerCombination[_sportMarkets[0]][_positions[0]][_sportMarkets[1]][_positions[1]][address(0)][0][address(0)][
-                0
-            ] += _sUSDPaid;
+            riskPerGameCombination[_sportMarkets[0]][_sportMarkets[1]][address(0)][address(0)][address(0)][address(0)][
+                address(0)
+            ][address(0)] += _sUSDPaid;
         } else if (_sportMarkets.length == 3) {
-            riskPerCombination[_sportMarkets[0]][_positions[0]][_sportMarkets[1]][_positions[1]][_sportMarkets[2]][
-                _positions[2]
-            ][address(0)][0] += _sUSDPaid;
+            riskPerGameCombination[_sportMarkets[0]][_sportMarkets[1]][_sportMarkets[2]][address(0)][address(0)][address(0)][
+                address(0)
+            ][address(0)] += _sUSDPaid;
         } else if (_sportMarkets.length == 4) {
-            riskPerCombination[_sportMarkets[0]][_positions[0]][_sportMarkets[1]][_positions[1]][_sportMarkets[2]][
-                _positions[2]
-            ][_sportMarkets[3]][_positions[3]] += _sUSDPaid;
-        }
+            riskPerGameCombination[_sportMarkets[0]][_sportMarkets[1]][_sportMarkets[2]][_sportMarkets[3]][address(0)][
```

Comment:
I am positive there is a way to rewrite this without 8 if..else statements.

Reply:
my previous push over-wrote the good one, probably didn't pull the right commit